### ISSUE TITLE
Suggestion: new user help tips

### ISSUE THREAD

**Issue by jonathandturner** [jmacdonald/amp#98]
The more I get into amp, the more I realise this thing is a serious editor with lots of features. Would be great to key users into this early on.

On way that might work (and please feel free to tweak as I'm still very much an amp-novice) is to put a little hint on the screen when you open a file, maybe at the bottom.  Say, before they touch any keys, you could have "Use '?' to bring up a tutorial" Or maybe `?` brings up the default keymap (which is another place I'm finding I'm learning what amp can do).

(This may relate to the resolution for https://github.com/jmacdonald/amp/issues/96)

**Comment by jonathandturner** [comment]
Yup, I've been spending quite a bit of time staring at that default keymap list to learn what amp can do. 

Think of these suggestions as ways to improve the new user experience. There's a lot available for users to do in amp, but currently you have to know where to look to find out how. A little tip at the beginning could go a long way. 

A tutorial that teachs you about tabbing through you files, creating new buffers, jump mode, the basics of edit, etc could be a great help.

It also helps to put forward "why use amp" early on, which helps users spend the time they need to in order to learn and get the benefit.

**Comment by jmacdonald** [comment]
These are great; please keep them coming!

It’s difficult to identify these kinds of discoverability issues once
you’ve gotten used to how Amp works, so while you’re still in this
“beginner’s” mindset, documenting these so we can address them going
forward is key.

I’ve thought about putting together a series of videos that walk through
some common workflows, and that touch on the features that support them.
It’d be great to complement that with a little bit of in-app documentation.

**Comment by Jackojc** [comment]
The splash screen for amp is a bit empty, maybe you could have some quick tips there like in vim?

**Comment by notramo** [comment]
Also, **automatic usage information for commands** would be good. When you start to type a command, then it would show a short description of the first selected completion candidate. See how the [Kakoune](http://kakoune.org/) editor does it.
